apps/auth-frontend/src/login/Login.tsx

```diff
@@ -7,6 +7,16 @@ import {
 } from '../auth0/web-auth';
 import { extractErrorMessage, WebAuthError } from '../auth0/errors';
 
+const getAppUrl = () => {
```

Comment:
We can move this to `config`. 
We can treat it as a constant, no need to keep invoking a function.

Reply:
I don't think so, it reads the current `window.location.search`

Reply:
I'm assuming the `auth-frontend` "entrypoint" needs the `redirect_uri` when loaded.
Haven't blocked the review because this isn't something i have strong opinions about :+1:

Reply:
Yeah sorry I noticed that after writing, so yeah it should work as a `constant`

Reply:
The problem is that it's very annoying to test. The only way is dynamically loading modules so that you have the opportunity to change `location` before loading the module. This is why I would pref reverting it

Reply:
Let's park this idea then :slightly_smiling_face: